lumigator/python/mzai/backend/backend/api/routes/experiments.py

```diff
@@ -60,3 +61,54 @@ def get_experiment_result_download(
     return ExperimentResultDownloadResponse.model_validate(
         service.get_job_result_download(experiment_id).model_dump()
     )
+
+
+####################################################################################################
+# "new" routes
+####################################################################################################
+# These "experiments_new" routes are not yet ready to be exposed in the OpenAPI schema,
+# because it is designed for the API when 'workflows' are supported
+# TODO: Eventually this route will become the / route,
+# but right now it is a placeholder while we build up the Workflows routes
+# It's not included in the OpenAPI schema for now so it's not visible in the docs
+@router.post("/new", status_code=status.HTTP_201_CREATED, include_in_schema=False)
+def create_experiment_id(
+    service: ExperimentServiceDep, request: ExperimentIdCreate
+) -> ExperimentResponse:
+    """Create an experiment ID."""
+    return service.create_experiment(request)
+
+
+@router.get("/new/all", include_in_schema=False)
```

Comment:
A general get should return all the applicable objects
```suggestion
@router.get("/new", include_in_schema=False)
```

Reply:
Funny you should mention that here 😆 , so I had that originally, but apparently the way FastAPI works is such that if you have a `get(/new `and` get(/new/{experiments_id}` endpoint, it assumes that you always are calling that second route. It's a different FastAPI behavior from if you have `get(/` and `get(/{experiments_id}` which differentiates fine. 🤷 

Since these routes are really just hidden routes that will be moved to replace the current experiments endpoints hopefully soon, I didn't spend too much time worrying about these route names in particular. And they're hidden from the schema so that people browsing the swagger/openAPI docs will not see them.

Reply:
There should be a way to structure this properly (possibly using the top level router). Please include a FIXME so we don't forget to do it in a later PR. I don't think we should block on this, but instead fix it in parallel.

Reply:
Will do 👍

Reply:
```suggestion
# TODO: FIXME this should not need the /all suffix.
# See further discussion https://github.com/mozilla-ai/lumigator/pull/728/files/2c960962c365d72e0714a16333884f0f209d214e#r1932176937
@router.get("/new/all", include_in_schema=False)
```